src/SimulationRunner.cc

```diff
@@ -153,6 +154,55 @@ SimulationRunner::SimulationRunner(const sdf::World *_world,
   // Load the active levels
   this->levelMgr->UpdateLevelsState();
 
+  // Load any additional plugins from the Server Configuration
+  this->LoadServerPlugins(this->serverConfig);
+
+  // If we have reached this point and no systems have been loaded, then load
+  // a default set of systems.
+  // Order:
+  // 1. File at IGN_GAZEBO_SERVER_CONFIG
+  // 2. File at ${IGN_HOMEDIR}/.ignition/gazebo/server.config
+  // 3. File at ${IGN_DATA_INSTALL_DIR}/server.config
+
+  if (this->systems.empty() && this->pendingSystems.empty())
+  {
+    ignmsg << "No systems loaded, loading defaults" << std::endl;
+
+    std::string envConfig;
+    ignition::common::env("IGN_GAZEBO_SERVER_CONFIG", envConfig);
+
+    std::string defaultConfig;
+    ignition::common::env(IGN_HOMEDIR, defaultConfig);
+    defaultConfig = ignition::common::joinPaths(defaultConfig, ".ignition",
+      "gazebo", "server.config");
```

Comment:
Similar to how #299 is adding a GUI config specific for log playback (`playback_gui.config`), I think we should add a `playback_server.config` to be loaded when `--record` is specified.

Reply:
I need to think about this to see if it will work with the current setup.